server/run.sh

```diff
@@ -1,4 +1,20 @@
 #!/bin/bash
+declare -a args
+test=false
+for i in $*; do
+	[ $i == "--test" ] && test=true || args[${#args}]=$i
+done
+if $test ; then
+	docker/generator.awk docker/dockerfile_cpsrv_template --test > docker/dockerfile_cpsrv_test
+	docker/generator.awk docker/docker-compose_template --test > docker-compose_test.yml
+	set -x
+	docker-compose -p cashpoints -f docker-compose_test.yml build $args
```

Comment:
No need in build step. `docker-compose up --build` for that case.
If you want to rebuild just do this one: `./run.sh --build`